src/test/ui/regions/regions-static-bound.stderr

```diff
@@ -0,0 +1,46 @@
+error[E0312]: lifetime of reference outlives lifetime of borrowed content...
+  --> $DIR/regions-static-bound.rs:6:5
+   |
+LL |     t
+   |     ^
+   |
+   = note: ...the reference is valid for the static lifetime...
```

Comment:
It seems like we're not looking at the bounds in the `where` clause to look for the source of `'static` requirement. Not necessary on this PR.